lighthouse/src/main.rs

```diff
@@ -85,6 +87,29 @@ fn build_profile_name() -> String {
         .to_string()
 }
 
+fn compiler_version() -> String {
+    let version = rustc_version::version_meta().expect("should be able to fetch compiler version");
```

Comment:
This will error if the command `rustc -vV` fails to run.

https://docs.rs/rustc_version/0.4.1/src/rustc_version/lib.rs.html#201
https://docs.rs/rustc_version/0.4.1/src/rustc_version/lib.rs.html#180

Is this run on compilation of on every run of LH?

Reply:
oh shit I think you're right, this is running at runtime 🤦‍♂️